api/namex/services/lookup/name_request_filing_actions.py

```diff
@@ -187,11 +187,11 @@ def get_actions(self, nr_type: str, entity_type_cd: str) -> Optional[dict]:
             URL: the location of the service
             learTemplate: the templat that can be used to create a draft filing.
         """
-        if nr_type in ['FR', 'CFR'] and entity_type_cd in ['FR', 'DBA', 'GP']:
-            nr_type = nr_type + '_' + entity_type_cd
+        if request_type_cd in ['FR', 'CFR'] and entity_type_cd in ['FR', 'DBA', 'GP']:
+            request_type_cd = request_type_cd + '_' + entity_type_cd
 
         # special case for continued in types that otherwise have the same NR type as regular companies
-        if entity_type_cd in ['C', 'CBEN', 'CCC', 'CUL']:
-            nr_type = nr_type + '_' + entity_type_cd
+        if request_action_cd == 'MVE':
+            request_type_cd = request_type_cd + '_' + entity_type_cd
```

Comment:
"If the action is MVE (cont in) then look up the entry with request type + entity type."
Eg,
request action = MVE
request type = CT
entity type = CR
look up =>"CT_CR"

Reply:
Namerequest UI should provide the correct entity type (eg, C instead of CR) for a MVE NR but I'm afraid that might break COLIN.

On the other hand, I don't think any of the continued in NRs worked before I added the cont in entries in the table above in the last major release.

Reply:
See my other comment regarding Namerequest UI.

I think the code fix could have been in either Namerequest UI or here. This PR fixes it here but will also work if Namerequest UI is changed :)